### ISSUE TITLE
`es-abstract` NOT Found

### ISSUE THREAD

**Issue by nyan2x** [airbnb/javascript#2104]
```
Dependencies:
eslint-config-airbnb 18.0.1 (latest)
   |- object.entries 1.1.0 (latest)
        |- es-abstract 1.12.0
```

- es-abstract > uses release version 1.14.0 which was deleted.
- object.entries > updated dependency (last Jul 25) but has not released a new version.
- object.entries > its older version (1.0.4) was from 3 yrs ago.

**Comment by ljharb** [comment]
v1.14.1, however, was published prior to v1.14.0 being unpublished - as such, semver should automatically fix it.

If you're using a lockfile, you've chosen to incur the obligation of updating transitive dependencies as needed. In this particular case, you can simply rerun the install command and it theoretically should update all dependencies to their latest available versions.

**Comment by nyan2x** [comment]
Thanks for the quick response!
